tests/integration/targets/ntnx_service_account_v2/tasks/main.yml

```
@@ -0,0 +1,12 @@
+---
+- name: Set module defaults
+  module_defaults:
+    group/nutanix.ncp.ntnx:
+      nutanix_host: "{{ ip }}"
+      nutanix_username: "{{ username }}"
+      nutanix_password: "{{ password }}"
+      validate_certs: "{{ validate_certs }}"
+  block:
+    - name: Import service_account_operations.yml
```

remove ignore errors